.catalog-onboard-pipeline.yaml

```diff
@@ -12,3 +12,9 @@ offerings:
         scc:
           instance_id: 1c7d5f78-9262-44c3-b779-b28fe4d88c37
           region: us-south
+      - name: fully-configurable
+        mark_ready: true
```

Comment:
I was just asked if we can set this to false for now so we can release the DA to community just with the fleets variation